### DISCUSSION TITLE
Release date of stable go client

### DISCUSSION THREAD

**Discussion by Wulfheart** [prisma/prisma-client-go#389] [Q&A] [answered]
Hello,

is there already an ETA for the first stable release of the go client?

Thank you.

**Comment by steebchen** [comment, marked as answer]
Hi there, thanks for asking!

Right now, there is no timeline of when we're going to publish a stable release.

Currently, we consider it an "early access product", meaning we plan to continuously support it by fixing bugs, incorporating new features, and keeping docs up to date.

At the same time, as the company focuses on the Javascript/Typescript client, the Go client gets only constrained resources, does not have feature parity with the JS/TS client and for now has its own documentation.

This mainly means that it's an early product which may have bugs or lacks in features.
However, from experience, whatever is supported right now is most likely guaranteed to work.
If you write Go client code, run it locally and it works, then it certainly will also work in production. This is because the actual database handling is done by the internal query engine written in rust (which is also used by the JS/TS client and thus considered stable).
So most likely if there's something not quite thought-through or missing, you will notice in development.

I hope this helps; if not, I'd be happy to answer any questions.
